tests/test_planloader.py

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+from substrait.planloader import planloader
+
+
+def test_main():
+    print(planloader.__file__)
+    print(dir(planloader))
+    testplan = planloader.load_substrait_plan('tests/tpch-plan01.json')
+    planloader.save_substrait_plan(testplan, 'myoutfile.splan', planloader.PlanFileFormat.TEXT.value)
```

Comment:
Round trip?  Or would the resulting be different?

Probably not very important since this is just testing that we can call the method and it doesn't explode.

Reply:
The round trip would be vulnerable to upstream changes in format.  As you note the connectivity test here is likely sufficient.